app/src/components/Header/Header.jsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Search from './../Search/Search';
+import Movie from './../Movie/Movie';
+import styles from './Header.scss';
+
+function Header(props) {
+  let content = '';
```

Comment:
I think ternary operator is more suitable here http://take.ms/V0lyN

Reply:
Done